### PR TITLE
Adds support of multisite sync policy feature check for RGW

### PR DESCRIPTION
**Pull request by udaysk23**
This PR adds support of multisite sync policy check for RGW
Jira link: https://issues.redhat.com/browse/RHCEPHQE-4648
Following is the log link of run:
http://pastebin.test.redhat.com/1075311

Also stand alone script is making tox failure, I have fixed that in this PR itslef rather than raising new one
Signed-off-by: uday kurundwade <ukurundw@redhat.com>

### CODE REVIEW

File: rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py [hunk 1]
```diff
@@ -76,6 +76,53 @@ def test_exec(config, ssh_con):
...
+                out = utils.exec_shell_cmd(cmd)
+                realm_name = period_list.get("realm_name")
+                cmd_realm = (
+                    f"radosgw-admin period update --rgw-realm={realm_name} --commit"
+                )
+                out = utils.exec_shell_cmd(cmd_realm)
+                cmd = "radosgw-admin sync group pipe remove --group-id=group1 --pipe-id=group1-pipe"
+                out = utils.exec_shell_cmd(cmd)
+                cmd = "radosgw-admin sync group flow remove --group-id=group1 --flow-id=group1-flow --flow-type=symmetrical"
+                out = utils.exec_shell_cmd(cmd)
```
Comment (TejasC88) [inline rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:118]: @udaysk23 Since we are creating and removing sync group pipe, whats the objective here?

Comment (udaysk23) [reply rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:118]: @TejasC88 Previously we had a bz on RGW crash when we tried to execute sync policy related commands.
We are executing these commands and checking there is crash or not

https://bugzilla.redhat.com/show_bug.cgi?id=2087986

Also above bz is part of close loop automtion

Comment (TejasC88) [reply rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:118]: @udaysk23 and I had a discussion , and we decided to take this code out to a different py file

**Comment by udaysk23** [comment]
Attached new log link from new code:
http://pastebin.test.redhat.com/1075490

File: rgw/v2/tests/s3_swift/test_multisite_sync_policy.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+from v2.tests.s3_swift import reusable
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+from v2.utils.utils import RGWService
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def get_multisite_info():
```
Comment (TejasC88) [inline rgw/v2/tests/s3_swift/test_multisite_sync_policy.py:41]: this method can be used by other also right , can we put it in reusables.py?

File: rgw/v2/tests/s3_swift/test_multisite_sync_policy.py [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    for i in range(len(period_list.get("period_map")["zonegroups"][0]["zones"])):
+        zone_name = period_list.get("period_map")["zonegroups"][0]["zones"][i]["name"]
+        zone_names = zone_names + zone_name
+        if i != len(period_list.get("period_map")["zonegroups"][0]["zones"]) - 1:
+            zone_names = zone_names + ","
+    realm_name = period_list.get("realm_name")
+    return zone_names, realm_name
+
+
+def update_commit():
```
Comment (TejasC88) [inline rgw/v2/tests/s3_swift/test_multisite_sync_policy.py:55]: same here

File: rgw/v2/tests/s3_swift/test_multisite_sync_policy.py (hunk 2 above)
Comment (udaysk23) [reply rgw/v2/tests/s3_swift/test_multisite_sync_policy.py]: Done

File: rgw/v2/tests/s3_swift/test_multisite_sync_policy.py (hunk 3 above)
Comment (udaysk23) [reply rgw/v2/tests/s3_swift/test_multisite_sync_policy.py]: Done

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**